core/src/mindustry/logic/LStatements.java

```diff
@@ -2169,6 +2169,10 @@ void rebuild(Table table){
             }else{
                 fieldst(table, "pan", pan, str -> pan = str);
             }
+            
+            table.row();
+            
+            fieldst(table, "check frame", checkFrame, str -> checkFrame = str);
```

Comment:
Would be nice to have this documented, with a better name. Maybe call it `limit` as in "limit to one sound per frame"? 

It could have a bundle description like this:

`playsound.limit = Limit this sound to only one playback per frame.\nOnly set this to false if absolutely necessary, e.g. for playing musical chords.`